repository/src/raw/tests.rs

```diff
@@ -448,3 +450,24 @@ async fn remote() {
         )]
     );
 }
+
+#[tokio::test]
+async fn reserved_state() {
+    let td = TempDir::new().unwrap();
+    let path = td.path();
+    let mut repo = init_repository_with_initial_commit(path).await.unwrap();
+
+    let (rs, _) = simperby_test_suite::generate_standard_genesis(10);
+
+    repo.checkout(MAIN.into()).await.unwrap();
```

Comment:
do we have main branch? what is it?

Reply:
Please note that this is a test for RawRepository, not DistributedRepository.

Reply:
right, understood